### PR TITLE
Automate Liquibase Extension build process - DAT-4607

### PR DESCRIPTION
**Pull request by mcred**
Currently extensions are built and release manually for each extension. This pull requests automates the release of the Liquibase Postgresql Extension completely from GitHub Actions with a few approval steps. Full workflow notes are available in `RELEASE.md`. 

There are two types of release: a release triggered by a version bump to liquibase core (#49 for example) and a manual release for a patch. This PR handles both scenarios, but the patch release requires additional manual steps as outlined in `RELEASE.md`.

Brief Workflow Overview:

*  Dependabot Opens a PR for liquibase core update
*  GitHub Actions runs the Unit Test
*  GitHub Actions labels the PR as `Extension Release Candidate :rocket:`
*  GitHub Actions runs the full Integration Test suite
*  User merges Pull Request into `main`
*  GitHub Actions builds signed artifact
*  GitHub Actions drafts a GitHub Release
*  User publishes GitHub Release
*  GitHub Actions uploads signed artifact to Sonatype Nexus
*  Sonatype Nexus will automatically release when staging checks are passed

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-1587) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by mcred** [comment]
Ignore the bogus version number that I created for testing. Example screenshot from during testing:
image

File: pom.xml [hunk 1]
```diff
@@ -82,120 +82,103 @@
...
+					<execution>
+						<id>integration-tests</id>
+						<phase>integration-test</phase>
+						<goals>
+							<goal>test</goal>
+						</goals>
+						<configuration>
+							<includes>
+								<include>**/*IT.*</include>
+								<include>**/*Tests.*</include>
```
Comment (molivasdat) [inline pom.xml:134]: Should this line be **/*Test.* and not Test**s**

File: pom.xml [hunk 2]
```diff
@@ -205,7 +188,7 @@
             <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
         </repository>
         <snapshotRepository>
-            <id>sonatype-nexus-snapshots</id>
+            <id>sonatype-nexus-staging</id>
```
Comment (molivasdat) [inline pom.xml:191]: Just asking.  I noticed this name was changed from snapshots to staging but the url remains to snapshots.  Is this correct?

Comment (mcred) [reply pom.xml:191]: It is. The Sonatype release manager manages which release goes where based on the `SNAPSHOT` suffix. More info [here](https://central.sonatype.org/publish/publish-maven/#distribution-management-and-authentication)

File: pom.xml (hunk 1 above)
Comment (mcred) [reply pom.xml]: Good catch. This has been updated.

**Review by molivasdat — APPROVED** [review]
From what I have reviewed looks good.

**Review by r2liquibase — APPROVED** [review]
I have no idea what I'm looking at. Why am I reviewing this? Ha!

**Pull request merged**